cmd/user-agent/src/pages/chapi/wallet/register/walletManager.js

```diff
@@ -21,7 +21,7 @@ export class WalletManager extends KeyValueStore {
     constructor() {
         super(`${dbName}-${metadataStore}`, metadataStore)
 
-        // TODO EDV will be used in future for these stores
+        // TODO SDS will be used in future for these stores #268
```

Comment:
This was an existing comment that I found. I'll need to investigate what these stores are and how to store them in SDS.